Resources/doc/security/disable_introspection.md

```diff
@@ -0,0 +1,19 @@
+Disable introspection
+=====================
+
+This bundle supports [webonyx/graphql-php validation rule to disable introspection queries](webonyx/graphql-php).
+
+Introspection is a mechanism for fetching schema structure. It is used by tools like GraphiQL for auto-completion, query validation, etc.
+
+It means that anybody can get a full description of your schema by sending a special query containing meta fields __type and __schema.
+
+If you are not planning to expose your API to the general public, it makes sense to disable this feature in production. By disabling, tools like GraphiQL won't work anymore.
+
+```yaml
+#app/config/config.yml
+overblog_graphql:
+    security:
+        disable_introspection: '%kernel.debug%'
```

Comment:
replace by `disable_introspection: false` since this example will disable introspection only in debug mode (dev environment by example)

Reply:
or just renaming the config entry `enable_introspection` will ease the usage of `%kernel.debug%` parameter. What do you think of that?

Reply:
Yes, i think that is probably the best solution. I will change this today or tomorrow and I will add the necessary change to have the introspection enabled by default to avoid any BC issues.